lib/plugins/aws/rollback.js

```diff
@@ -44,45 +41,41 @@ class AwsRollback {
     const deploymentPrefix = this.provider.getDeploymentPrefix();
     const prefix = `${deploymentPrefix}/${serviceName}/${stage}`;
 
-    return this.provider
-      .request('S3', 'listObjectsV2', {
-        Bucket: this.bucketName,
-        Prefix: prefix,
-      })
-      .then((response) => {
-        const deployments = findAndGroupDeployments(response, deploymentPrefix, serviceName, stage);
+    const response = await this.provider.request('S3', 'listObjectsV2', {
+      Bucket: this.bucketName,
+      Prefix: prefix,
+    });
+    const deployments = findAndGroupDeployments(response, deploymentPrefix, serviceName, stage);
 
-        if (deployments.length === 0) {
-          const msg = "Couldn't find any existing deployments.";
-          const hint = 'Please verify that stage and region are correct.';
-          return BbPromise.reject(`${msg} ${hint}`);
-        }
+    if (deployments.length === 0) {
+      const msg = "Couldn't find any existing deployments.";
+      const hint = 'Please verify that stage and region are correct.';
+      throw new Error(`${msg} ${hint}`);
```

Comment:
That's a good pick.

Still it feels as a _user_ error (not an error on our side), and in that case we communicate those with `ServerlessError` and we also provide a code (search for `ServerlessError` and it should bring tons of such cases)